src/storage/src/table/storage_table.rs

```diff
@@ -368,7 +368,9 @@ impl<S: StateStore, RS: RowSerde, const T: AccessType> StorageTableBase<S, RS, T
 
         tracing::trace!(target: "events::storage::storage_table", "compute vnode: {:?} key {:?} => {}", row, indices, vnode);
 
-        self.check_vnode_is_set(vnode);
+        if !indices.is_empty() {
```

Comment:
May leave a FIXME here showing that this is a workaround for local agg with a lack of a vnode builder. :)